tests/transports-governance/test_virtual_keys_crud.py

```
@@ -0,0 +1,928 @@
+        # Update concurrently with different descriptions
+        def update_vk(index):
+            update_data = {"description": f"Updated by thread {index}"}
+            response = governance_client.update_virtual_key(vk_id, update_data)
+            return response, index
+
+        with ThreadPoolExecutor(max_workers=5) as executor:
+            futures = [executor.submit(update_vk, i) for i in range(5)]
+            results = [future.result() for future in futures]
+
+        # All updates should succeed (last one wins)
+        for response, index in results:
```

Ruff (0.12.2)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)